packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -68,7 +73,22 @@ import { RootDataObject } from "./rootDataObject";
  * ```
  */
 export interface IFluidContainerEvents extends IEvent {
-    (event: "connected" | "dispose" | "disconnected" | "saved" | "dirty", listener: () => void): void;
+    /**
+     * **connected** & **disconnected** events reflect connection state changes against the (delta)
+     * service acknowledging ops/edits.
+     */
+    (event: "connected" | "disconnected", listener: () => void): void;
+    /**
+     * **saved** event is raised when all local changes/edits have been acknowledged by the service.
```

Comment:
Any better terminology we can use other than "acknowledged" ?

Reply:
"sequenced" maybe?  When we see the ops we sent get sequenced, we frequently refer to that as an "ack" so either would have precedence in our terminology.

Reply:
OK - I'll stick to "ack"/"acknowledged" in that case, since it's a bit more concrete term.